src/index.html

```diff
@@ -895,6 +896,22 @@ <h5>Archive process</h5>
                                         </div>
                                     </li>
                                 </div>
+                                <div id="what-data-is-brokered-to-dbsnp">
+                                    <li class="accordion-item is-active" data-accordion-item>
+                                        <a class="accordion-title"><h5>Which data is brokered to DBSNP?</h5></a>
+                                        <div class="accordion-content" data-tab-content>
+                                            <p>Any variation dataset containing human genotype or allele frequency will be brokered to dbSNP for variant accessioning an assignment of RS ids.</p>
+                                        </div>
+                                    </li>
+                                </div>
+                                <div id="how-do-we-deal-with-chromosome-name-diversity">
+                                    <li class="accordion-item is-active" data-accordion-item>
+                                        <a class="accordion-title"><h5>How do we deal with Chromosome name diversity?</h5></a>

```

Comment:
Maybe:
```suggestion
                                        <a class="accordion-title"><h5>How do we deal with chromosome or contig name diversity?</h5></a>

```
Just because we kind of use both terms interchangeably in the text...

Reply:
updated